_pages/index.md

```diff
@@ -56,7 +56,7 @@ report:
   lead: |-
     If you believe that you or someone else experienced unlawful discrimination, you can report a disability rights violation.
   content: |-
-    <strong>Pick a topic to find out how the ADA works.</strong> We’ll add more soon
+    <strong>Pick a topic to find out how the ADA works.</strong> We’ll add more soon.
   steps:
```

Comment:
Oops, this must have been a copy/paste error on my part - this content block can be removed since it is not actually being used by the reporting section.

Reply:
I was wondering about that. I'll push another commit.